Tests/scripts/slack_notifier.py

```diff
@@ -40,9 +40,9 @@ def options_handler():
 
 
 def get_attachments(build_url, env_results_file_name):
-    content_team_fields, content_fields, failed_tests = get_fields()
+    content_team_fields, content_fields, failed_tests, failed_unittests = get_fields()
     color = 'good' if not failed_tests else 'danger'
-    title = 'Content Build - Success' if not failed_tests else 'Content Build - Failure'
+    title = 'Content Build - Success' if not (failed_tests or failed_unittests) else 'Content Build - Failure'
```

Comment:
shouldn't the operator in the condition be `and` ?

Reply:
No, because if one of the failures won't exist and the other yes, it will return a success with the `and` operator.